docs/bug-triage.md

```diff
@@ -0,0 +1,88 @@
+# Bug triage process
+
+## Actions
+
+When triaging a bug in most cases there is a single action required. Actions can be
+executed on an GitHub Issue either by adding the label `action-<action name>` or
+adding `~<action-name>` as the last line of the comment.
+
+For example if an issue is missing information you could comment:
+
+```
+There is not enough information here, and I'm not following the steps-to-reproduce.
+
+~missing-info
+```
+
+You could also for example just add the label `action/priority-regression` to an issue.
+
+## Triaging bugs
+
+### Verify the issue
+
+The first step is to verify if the issue is a valid issue following these questions:
+
+![Bug triaging - Verify!](bug-triage-verify.svg "Bug Triage - Verify")
+
+If an issue is not valid add a comment with some explanation, and add `~<action-name>` as the last line of the comment
+to trigger corresponding action. 
+
+In cases where it is clear that no additional comment is needed you can just add the `action/<action-name>` label. For
+example if the description only states `It doesn't work` then there's not much point in explaining what information is
+missing.
+
+### Prioritize the issue
+
+Second step is to prioritize the bug depending on how common the use-case is, if it's a regression, 
+or not blocking anything, for example a typo:
+
+![Bug triaging - Prioritize!](bug-triage-prioritize.svg "Bug Triage - Prioritize")
+
+When selecting the priority for an issue add the `action-<action-name>` label to the issue.
+
+
+## Missing information
+
+Bugs with insufficient information are assigned the labels `status/missing-info` and `status/auto-expire`.
+
+If the original reporter provides additional information the issue is automatically moved back to triage. 
+Otherwise, if the reporter does not provide additional information within 14 days the issue is automatically closed.
+
+To prevent an issue with missing information to be automatically closed, remove the `status/auto-expire` label.
+
+
+## Low and normal priority issues
+
+In most cases these are not issues we will fix, and we will look for contributions to resolve them. 
+As such the team does not actively monitor issues with these priorities.
+
+The bot will automatically bump the priority based on reactions added to the issue description:
+
+* Low priority bumped to normal if there are 10 or more reactions
+* Normal priority bumped to important if there are 20 or more reactions
+
+To prevent an issue from being automatically bumped, remove the `status/auto-bump` label.
+
+If there are no updates to low or normal priority issues they will be automatically closed:
+
+* Low priority are closed after 90 days
+* Normal priority are closed after 180 days
+
+To prevent an issue from being automatically closed, remove the `status/auto-expire` label.
```

Comment:
Should we mention something like this in this section:
```
The label `help wanted` is automatically added to these issues signalling we welcome a community contribution for them. Consider also adding the label `good first issues` if it suits the project's newcomers. In this case, you may include some hints on how to fix this issue in your comment, for example, class names where you expect the changes, where to add a test, etc. 
```

Reply:
Expanded this a bit to mention that we are looking for contributions by adding the `help wanted` label